hieradata/common.yaml

```diff
@@ -769,6 +769,11 @@ govuk_ci::master::ci_agents:
   ci-agent-5:
     agent_hostname: 'ci-agent-5.ci'
     labels: 'mongodb-3.2 ci-agent-5 elasticsearch'
+  ci-agent-6:
+    agent_hostname: 'ci-agent-6.ci'
+    labels: 'mongodb-3.2 ci-agent-6 xenial'
```

Comment:
if we're now making this exclusive we probably do not want mongodb-3.2 as a label